riscv/execute.cc

```diff
@@ -215,7 +215,7 @@ static inline reg_t execute_insn(processor_t* p, reg_t pc, insn_fetch_t fetch)
 
 bool processor_t::slow_path()
 {
-  return debug || state.single_step != state.STEP_NONE || state.debug_mode;
+  return i_trace | d_trace | debug || state.single_step != state.STEP_NONE || state.debug_mode;
```

Comment:
This '|' instead of '||' seems like logical bug (but it will work). Below it is correct.